src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+import duke.command.CommandHandler;
+import duke.input.InputHandler;
+import duke.input.InputParser;
+import duke.output.DisplayHandler;
+import duke.storage.ListHandler;
```

Comment:
Perhaps the ordering of the import statements could be reworked to align with the instantiation and dependencies below just before the while loop and inside of it as well ?